### PR TITLE
Log queue error on startup, do not show crash endpoint out of debug

### PR DESCRIPTION
**Pull request by RubenGeo**
[AB#31640](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/31640) 

## Describe your changes

- Log handling job startup errors to azure if it fails
- Do not show endpoint out of debug

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have added tests wherever relevant
- [ ] I have made sure that all automated checks pass before requesting a review
- [ ] I do not need any deviation from our PR guidelines

### CODE REVIEW

File: k6/tests/retryFailedJobsOnStartupDuringQueueProcessing.js [hunk 1]
```diff
@@ -84,6 +84,4 @@ export default function () {
       return r.status == 200;
     },
   });
-
-  sleep(1);
```
Comment (RubenGeo) [inline k6/tests/retryFailedJobsOnStartupDuringQueueProcessing.js:88]: What there a point to this sleep? @jannisvisser

Comment (jannisvisser) [reply k6/tests/retryFailedJobsOnStartupDuringQueueProcessing.js:88]: don't know, but probably not.

File: services/121-service/src/scripts/scripts.controller.ts [hunk 2]
```diff
@@ -193,6 +199,7 @@ export class ScriptsController {
     summary:
       'WARNING: Kills 121-service. Only works in DEBUG-mode. Only used for testing purposes.',
   })
+  @ApiExcludeEndpoint(!DEBUG)
```
Comment (jannisvisser) [inline services/121-service/src/scripts/scripts.controller.ts:202]: nice, this just means it doesn't show up in swagger? or it doesn't exist at all?

File: services/121-service/src/queue-registry/queue-registry.service.ts [hunk 3]
```diff
@@ -76,8 +79,8 @@ export class QueueRegistryService implements OnModuleInit {
         return await this.retryFailedJobs();
       })
       .catch((err) => {
-        // ##TODO: handle differently
         console.error('Error in retryFailedJobs: ', err);
+        this.azureLogService.logError(err, true);
```
Comment (jannisvisser) [inline services/121-service/src/queue-registry/queue-registry.service.ts:83]: did you check this functionally?

**Review by jannisvisser — APPROVED** [review]
see some questions. conditionally approved, based on that.

File: services/121-service/src/queue-registry/queue-registry.service.ts [hunk 4]
```diff
@@ -71,17 +74,23 @@ export class QueueRegistryService implements OnModuleInit {
...
+      console.error('scheduleRetryFailedJobs', err);
+    });
+  }
+
+  private async scheduleRetryFailedJobs(): Promise<void> {
+    try {
+      await new Promise((resolve) => setTimeout(resolve, 5000));
+      await this.retryFailedJobs();
+    } catch (err) {
+      console.error('Error in retryFailedJobs: ', err);
```
Comment (jannisvisser) [inline services/121-service/src/queue-registry/queue-registry.service.ts:88]:
```suggestion
      console.error('Error in scheduleRetryFailedJobs: ', err);
```

**Review by jannisvisser — APPROVED** [review]

**Pull request merged**